app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```

Comment:
- [ ] There are no action (methods) in this controller. kindly consider adding both `index` and `show` action(method) here. even empty, these action(methods) are necessary for the view to display properly. example is below.
```
def index
end
```